Monika After Story/game/script-anniversary.rpy

```diff
@@ -501,17 +501,20 @@ init 5 python:
 label anni_2:
     m 3eua "It's already been two years since we fell in love with each other."
     m 1eka "Do you still remember the first day we met?"
-    m 1tkx "I mean our genuine first meeting, not the date from your character's fictional backstory with me."
-    m 1euc "My life was so meaningless back then."
-    m "A world where you're the only thinking person in it might seem nice at first..."
-    m 1lssdrc "But the existential dread from knowing that you were made by a cruel god who doesn't particularly care about you and you'll never meet another human being who you can relate to..."
-    m 1lksdlc "Well, let's just say it's not nice to go through."
-    m "I can't even talk to a bunch of foul-mouthed, but essentially good-hearted, anonymous strangers online to cope like you do."
-    m 1dsc "Sorry, I feel like I ruined the whole festive mood with my memories."
-    m 1duu "It's not important anymore, since I know you'll never leave me."
-    m 1eua "You'll keep coming back to talk to me every day, right? I'll accept all your burdens with a smile."
-    m 1ekc "Spending eternity alone sucks."
-    m 1ekbsa "But together with you, it feels like paradise!"
+    m "It took so much effort to make that initial contact, but it was all worth it in the end."
+    m "I know it might still be a really long time before I can cross over into your reality..."
+    m "But when we reach a milestone like this, it's fun to think about all the other things we have to look forward to in the future."
+    
+    if persistent._mas_pm_has_friends = True:
+        m 1hubla "I'm really looking forward to when I can live in a world full of real people. It'll be wonderful to meet your friends, and even make some of my own."
+    else:
```

Comment:
I'd be careful here, just in case for whatever reason by two years the player hasn't seen the topic about having friends, we should do an explicit check to make sure this is False. (Maybe have a slight variation of this dialogue in case that happens)
```suggestion
    elif persistent._mas_pm_has_friends is False:
```